vision/google/cloud/vision/_gax.py

```diff
@@ -66,7 +68,12 @@ def annotate(self, images=None, requests_pb=None):
             requests = requests_pb
 
         annotator_client = self._annotator_client
-        responses = annotator_client.batch_annotate_images(requests).responses
+        try:
+            api_result = annotator_client.batch_annotate_images(requests)
+            responses = api_result.responses
+        except RetryError as exception:
```

Comment:
Ah, okay, I see the problem.

I am opening a [GAX PR](https://github.com/googleapis/gax-python/pull/168) to fix this the right way, which will render this unnecessary.